db/migrate/20221201194057_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] It looks like there is an error in the comments migration file. Take a look at the image below.

![comments](https://user-images.githubusercontent.com/53124548/205258409-f5dba131-98db-44e3-bc01-4f6e0932751a.png)

Reply:
- Thanks for your review, however, @elisha2kyakpo1  can you be more clear on that because , for me, I did not get an error. What did you do to get the error?

Reply:
@elisha2kyakpo1  Take a look at this picture 

![Screenshot from 2022-12-02 12-53-34 (1)](https://user-images.githubusercontent.com/80458199/205283131-8dd6c36d-0796-448b-beba-a7951313248d.png)